inst/rmarkdown/templates/outbreak/skeleton/skeleton.Rmd

```diff
@@ -291,7 +300,6 @@ The case fatality ratio among those with known outcomes is below
 
 ```{r identify_deaths}
 # define your input variables in advance to pass through CFR function
-# CHANGE TO DPLYR SYNTAX??
 known_status <- linelist_cleaned[!is.na(linelist_cleaned$outcome), ]
```

Comment:
these bits are what i meant should be changed to dplyr filter rather than square brackets... thoughts?

Reply:
Ah! I though I had removed it! Will remove.